packages/@aws-cdk/aws-ecs/lib/scheduled-ecs-task.ts

```diff
@@ -0,0 +1,98 @@
+import events = require('@aws-cdk/aws-events');
+import cdk = require('@aws-cdk/cdk');
+import { ICluster } from './cluster';
+import { ContainerImage } from './container-image';
+import { Ec2EventRuleTarget } from './ec2/ec2-event-rule-target';
+import { Ec2TaskDefinition } from './ec2/ec2-task-definition';
+import { AwsLogDriver } from './log-drivers/aws-log-driver';
+
+export interface ScheduledEc2TaskProps {
+  /**
+   * The cluster where your service will be deployed.
+   */
+  readonly cluster: ICluster;
+
+  /**
+   * The image to start.
+   */
+  readonly image: ContainerImage;
+
+  /**
+   * Number of desired copies of running tasks.
+   *
+   * @default 1
+   */
+  readonly desiredTaskCount?: number;
+
+  /**
+   * The hard limit (in MiB) of memory to present to the container.
+   *
+   * If your container attempts to exceed the allocated memory, the container
+   * is terminated.
+   *
+   * @default 256
+   */
+  readonly memoryLimitMiB?: number;
+
+  /**
+   * The schedule expression used to configure the event rule.
+   */
+  readonly scheduleExpression?: string;
+
+  /**
+   * Environment variables to pass to the container.
+   *
+   * @default No environment variables
+   */
+  readonly environment?: [{ [key: string]: string }];
+
+  /**
+   * Whether to create an AWS log driver to enable logging.
+   *
+   * @default true
+   */
+  readonly enableLogging?: boolean;
+}
+
+/**
+ * A scheduled Ec2 task that will be initiated off of cloudwatch events.
+ */
+export class ScheduledEc2Task extends cdk.Construct {
+  constructor(scope: cdk.Construct, id: string, props: ScheduledEc2TaskProps) {
+    super(scope, id);
+
+    // Create a Task Definition for the container to start
+    const taskDefinition = new Ec2TaskDefinition(this, 'ScheduledTaskDef');
+    taskDefinition.addContainer('ScheduledTaskContainer', {
+      image: props.image,
+      memoryLimitMiB: props.memoryLimitMiB !== undefined ? props.memoryLimitMiB : 256,
+      logging: props.enableLogging ? this.createAWSLogDriver(this.node.id) : undefined,
+    });
+
+    // An EventRule that describes the event trigger (in this case a scheduled run)
+    const rule = new events.EventRule(this, 'ScheduledTaskRule', {
+      scheduleExpression: props.scheduleExpression,
+    });
+
+    // Use Ec2TaskEventRuleTarget as the target of the EventRule
+    const target = new Ec2EventRuleTarget(this, 'ScheduledEventRuleTarget', {
+      cluster: props.cluster,
+      taskDefinition,
+      taskCount: props.desiredTaskCount
```

Comment:
It says the default is  1 in the props docstring but it doesn't appear to be set here?